src/ray/protobuf/node_manager.proto

```diff
@@ -331,7 +321,17 @@ message GetObjectsInfoReply {
   int64 total = 2;
 }
 
-message GetResourceLoadRequest {}
+message GetResourceLoadRequest {
+  // The last seen version, -1 indicating no resources update has been seen.
+  // If the current version on the server end is not newer than this, the server
+  // will skip populating the resources view of the node, but only adding
+  // the newest load information.
+  // NOTE(rickyx): We could also do lightweight load reporting, but since we
+  // are only updating load info less frequentlys, 1 sec by default by
+  // gcs_pull_resource_loads_period_milliseconds, it's ok to send the entire load
+  // info, which is usally small.
+  int64 last_seen_version = 1;
```

Comment:
This is needed so that we could skip populating resources when getting the load (similar to lightweight resources broadcasting). 

However, since this is GCS polling raylets, it needs to tell the raylet which is the most recent version of resources so the raylet could decide if data needs to be populated.